recipes/msix/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class MsixConan(ConanFile):
+    name = "msix"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/msix-packaging"
+    description = "An SDK for creating MSIX packages"
+    topics = ("msix", "sdk", "packaging", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "crypto_lib": ["crypt32", "openssl"],
```

Comment:
`crypt32` is only valid on Windows, so add a check in `validate` to only allow `crypt32` on Windows.

Reply:
[Fixed](https://github.com/conan-io/conan-center-index/pull/6475/commits/0e77bd7ec9eab25036be96cd42fb99bc5297c326)